openadapt/config.py

```diff
@@ -32,6 +32,7 @@
 PERFORMANCE_PLOTS_DIR_PATH = (DATA_DIR_PATH / "performance").absolute()
 CAPTURE_DIR_PATH = (DATA_DIR_PATH / "captures").absolute()
 VIDEO_DIR_PATH = DATA_DIR_PATH / "videos"
+DATABASE_FILE_PATH = DATA_DIR_PATH / "openadapt.db"
```

Comment:
Please update `DB_URL` to re-use this. Also please call `absolute()` on this.

Reply:
Done here: d82a77d7550d7cc3a903f718d259b14c038f961d